src/Components/ClickAndEdit.js

```diff
@@ -27,76 +38,95 @@ export default function ClickAndEdit({
     if (e?.key === "Enter") e.preventDefault();
   }
 
-  return (
-    <div>
-      {!editDesc ? (
-        <Typography
-          sx={{
-            fontFamily: styling?.fontFamily ?? "interMedium",
-            fontSize: styling?.fontSize ?? "1rem",
-            color: data?.length > 0 ? "unset" : theme.palette.text.secondary,
-            pb: styling?.pb ?? 1,
-            pl: 0,
-            ml: 0,
-            cursor: canEdit ? "pointer" : "unset",
-          }}
-          tabIndex={canEdit ? "0" : "-1"}
-          onClick={canEdit ? handleDescToggle : undefined}
-          onKeyDown={(e) => {
-            if (e.key === "Enter") handleDescToggle(e);
-          }}
-        >
-          {data?.length > 0 ? data : canEdit ? placeholder : ""}
-        </Typography>
-      ) : (
-        <div
-          style={{
-            display: "flex",
-            flexDirection: "column",
-            alignItems: "center",
-          }}
-        >
-          <TextField
-            name="Desc"
-            id="Desc"
-            size="small"
-            variant="filled"
-            autoComplete="off"
-            color="text"
-            placeholder={placeholder}
-            autoFocus
-            multiline
-            fullWidth={true}
-            value={editedDesc}
-            onClick={(e) => e.preventDefault()}
-            onChange={(e) => {
-              setEditedDesc(e.target.value);
-            }}
-            sx={{ mb: 2 }}
-            InputProps={{
-              style: {
-                padding: "10px 10px 10px 5px",
-              },
-            }}
-            inputProps={{
-              style: {
-                maxWidth: "none",
-              },
-            }}
-          ></TextField>
-          <Button
-            size="small"
-            variant="outlined"
-            color="inherit"
-            onClick={(e) => {
-              saveDesc();
-              handleDescToggle();
+  if (data) {
```

Comment:
I can change my name to an empty string, which then won't allow me to edit further since the empty string is falsy.  Should we be more specific and check `if (data !== undefined)` perhaps?

Reply:
Also, we should be able to get rid of this check once we apply the fix I mentioned to implement ghost display for `ProfileUserBanner`.  Maybe leave a TODO?